x-pack/plugins/secops/server/graphql/scalar_date/resolvers.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { GraphQLScalarType, Kind } from 'graphql';
+
+export const dateScalar = new GraphQLScalarType({
+  name: 'Date',
+  description:
+    'Represents a Date for either an ES formatted date string or epoch string ISO8601 formatted',
+  serialize(value): string {
+    return Number.isNaN(Date.parse(value)) ? new Date(value).toISOString() : value;
+  },
+  parseValue(value) {
+    return value;
```

Comment:
maybe here we can do the conversion to make sure that we always get epoch value on the server side

Reply:
next time ;)